presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -109,6 +109,10 @@ Operations
 
     Returns the bounding rectangular polygon of a geometry.
 
+.. function:: ST_EnvelopeAsPts(Geometry) -> Geometry
+
+    Returns the lower left and upper right corners of bounding rectangular polygon of a geometry.
```

Comment:
Let's clarify that return type is an array.

`Returns an array of two points: the lower left and upper right corners of the bounding rectangular polygon of a geometry. Returns null if input geometry is empty.`

Reply:
Done